iphone/Classes/TiUIListView.m

```diff
@@ -2433,6 +2339,61 @@ - (void)handleTap:(UITapGestureRecognizer *)tapGestureRecognizer
   // Never called
 }
 
+#pragma mark - DimmingView Manage
+
+- (void)createDimmingView
+{
+  if (dimmingView == nil) {
+    dimmingView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, self.frame.size.width, self.frame.size.height)];
+    dimmingView.backgroundColor = [UIColor blackColor];
+    dimmingView.alpha = .2;
+    UITapGestureRecognizer *tapGesture = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(dismissSearchController)];
+    [dimmingView addGestureRecognizer:tapGesture];
+  }
+}
+
+- (void)showDimmingView
+{
+  dimmingView.frame = CGRectMake(0, 0, self.frame.size.width, self.frame.size.height);
+  if (!dimmingView.superview) {
+    [self addSubview:dimmingView];
+    [self bringSubviewToFront:dimmingView];
+  }
+}
+- (void)hideDimmingView
+{
+  [dimmingView removeFromSuperview];
+}
+
+- (void)dismissSearchController
+{
+  [searchController setActive:NO];
+}
+
+- (void)keyboardWillChangeFrame:(NSNotification *)notification
+{
+  NSDictionary *userInfo = [notification userInfo];
```

Comment:
Unused property. Either remove or change the below statement to use it. Same for `keyboardDidChangeFrame` and the two implementations for the TiUITableView.

Reply:
Change the statement to use the property. 
We are managing the view of SearchController, due to that there is some glitch in orientation change. In slow animation (debug), it is more visible. In normal orientation it will not be visible that much. I'll create a ticket to work on this issue.